pkg/context/txnclient/client.go

```diff
@@ -141,7 +147,10 @@ func (c *Client) buildCredential(anchor string, refs []*operation.Reference, ver
 		Issued: &util.TimeWithTrailingZeroMsec{Time: time.Now()},
 	}
 
-	// TODO: Sign VC here
+	signedVC, err := c.TxnSigner.Sign(vc)
```

Comment:
where does the domain property get set?

Reply:
Do we need it? I will add it if we do (as server configuration parameter?). 
I also have to do to configure:
1. issuer
2. anchor credential context

Reply:
yes - e.g., LD proof must contain the domain property. this is how the backing ledger becomes discovered.

Reply:
ok, I will add it together with issuer in my next PR.